.github/workflows/actions.yml

```diff
@@ -7,6 +7,10 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
```

Comment:
shouldnt it be yarn?

Reply:
We use npm here, and if it's working I ain't gonna break it 😅